README.md

```diff
@@ -2,18 +2,41 @@
 
 Typetura is a tool to make fluid typography easy. Here is a [demo of typetura in action](http://typetura-js.netlify.com). It enables you to control the font size, line height, margins, padding, variable font settings, and anything that can be animated. The difference is that the keyframes are applied across screen sizes as opposed to time.
 
-### Installing typetura on your website
+# Installing typetura on your website
 
-* Add `typetura.js` where you put yout scripts.
-* Start styling with CSS.
+```bash
+npm install --save typeturajs
+```
+
+## Via script tag
+
+```html
+<script src="./node_modules/typetura.js/js/typetura.min.js"></script>
+```
+
+## commonJS
```

Comment:
Before someone could just add the script and the CSS would just work without them touching the JS. I think there is value in this configuration being accessable to those who don’t know or aren’t comfortable with JS.

Way of selecting the root element here: https://github.com/scottkellum/typetura.js#work-off-the-width-of-an-element-and-more